packages/@o3r/components/schematics/ng-add/index.ts

```diff
@@ -13,11 +13,15 @@ export function ngAdd(): Rule {
       const { ngAddPackages, getO3rPeerDeps, removePackages } = await import('@o3r/schematics');
       const depsInfo = getO3rPeerDeps(path.resolve(__dirname, '..', '..', 'package.json'));
 
-      return chain([
+      const rule = chain([
         removePackages(['@otter/components']),
         ngAddPackages(depsInfo.o3rPeerDeps, { skipConfirmation: true, version: depsInfo.packageVersion, parentPackageInfo: depsInfo.packageName })
       ]);
 
+      context.logger.info(`The package ${depsInfo.packageName!} comes with debug mechanism`);
+      context.logger.info('Get information on https://github.com/AmadeusITGroup/otter/tree/main/docs/components/COMPONENT_STRUCTURE.md#Runtime-debugging');
```

Comment:
To me, your get information on/about a subject. It looks like you are getting information about the link, but I do not know how to fix it.
@sdo-1A could you please help :) ?